1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,9 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy = store.filter((e) => e.price < money).map((e) => {
```

Comment:
The condition should also be true if e.price is equal to money. Could you change the condition please?